src/apps/schema/src/app/components/CreateModelDialogue.tsx

```diff
@@ -138,7 +132,9 @@ export const CreateModelDialogue = ({ onClose, modelType = "" }: Props) => {
               canonicalTagMode: 1,
               metaLinkText: model.label,
               metaTitle: model.label,
-              parentZUID: model.parentZUID || "0",
+              parentZUID: model.parentZUID.startsWith("7-")
```

Comment:
This seems like it would prevent models from being nested to other valid models.

Reply:
okay. maybe a detail worth of a comment so we remember the expectation/behavior combo when we come back to this code